src/docker.rs

```diff
@@ -154,18 +154,48 @@ pub fn run(target: &Target,
         .args(&["-v", &format!("{}:/rust:ro", rustc::sysroot(verbose)?.display())])
         .args(&["-v", &format!("{}:/target", target_dir.display())])
         .args(&["-w", "/project"])
-        .args(&["-it", &image(toml, target)?])
+        .args(&["-it", &image(toml, target, verbose)?])
```

Comment:
@Dylan-DPC was this verbose (and also below) removed at some point ?

Reply:
nope. just checked the history it wasn't changed for the last 2 years at least